code/modules/mob/living/carbon/human/species/station/traits_vr/positive.dm

```diff
@@ -42,13 +42,13 @@
 	name = "Darksight"
 	desc = "Allows you to see a short distance in the dark."
 	cost = 1
-	var_changes = list("darksight" = 3)
+	var_changes = list("darksight" = 3, "flash_mod" = 2.0)
```

Comment:
flash_mod is way too high... Vanilla species only use 1.1 and 1.2. 2.0 and 3.0 are way over the top. I would recommend 1.2 and 1.4, respectively.

Reply:
The previous photosensitive trait was 2.

Reply:
Perhaps 1.5 and 2.0, then?

Reply:
How bad is a 2.0 and 3.0 flash mod in practice? Darksight is a really good bonus so if it's as simple as 2x/3x as long stun duration it makes sense. If anything it gives me an excuse to RP my darksight char's as disliking bright areas.

Reply:
Live the Riddick lifestyle. Welding gogs 24/7.

Reply:
> How bad is a 2.0 and 3.0 flash mod in practice

0 because nobody gets flashed which is why it was the meta